spec/50-privacy.md

```diff
@@ -19,3 +19,9 @@ The  `tracestate` field may contain any opaque value in any of the keys. The mai
 Tracing systems MUST NOT include any personally identifiable information in the `tracestate` header.
 
 Platforms and tracing systems extremely sensitive to personal information exposure MAY implement selective removal of values corresponding to the unknown keys. This mutation of the `tracestate` field is not forbidden, but highly discouraged. As it defeats the purpose of this field for allowing multiple tracing systems to collaborate.
+
+## Other risks
+
+In implementations where headers `traceparent` and `tracestate` used as response
```

Comment:
Grammar changes:

In implementations where headers `traceparent` and `tracestate` are included in responses, these values may inadvertently be passed to cross-origin callers. Implementations should ensure that they only include these response headers when responding to systems that participated in the trace.